packages/core/src/runtime/utils/hashClassName.ts

```diff
@@ -6,10 +6,11 @@ export interface HashedClassNameParts {
   value: string;
   pseudo: string;
   media: string;
+  layer: string;
   support: string;
 }
 
-export function hashClassName({ media, property, pseudo, support, value }: HashedClassNameParts): string {
+export function hashClassName({ media, layer, property, pseudo, support, value }: HashedClassNameParts): string {
```

Comment:
Why is the `layer` added as an argument and then not used in the hash?

Reply:
No reason. I started adding 'layer' everywhere I saw media.

Reply:
Seems i missed one :)

Reply:
added it into hashing